docs/source/tools/assistant.rst

```diff
@@ -127,15 +126,15 @@ Here is what each field means:
 - ``dependencies``: library-dependencies of this project. See :doc:`/daml/reference/packages`.
 - ``data-dependencies``: Cross-SDK dependencies of this project See :doc:`/daml/reference/packages`.
 - ``module-prefixes``: Prefixes for all modules in package See :doc:`/daml/reference/packages`.
-- ``scenario-service``: settings for the scenario service
+- ``script-service``: settings for the script service
```

Comment:
ah nice to see we're starting to rename the service

Reply:
yeah I’m not touching the code for now because who cares but at least in user-facing stuff it seems worth renaming things :)

Reply:
yup!